.travis.yml

```diff
@@ -38,35 +51,39 @@ jobs:
     script: make spelling
   - &FLAKE
     name: flake
+    env: {}
     python: "3.6"
     install:
     - pip install -r tests/requirements.txt
     script:
     - make flake
   - <<: *FLAKE
     python: "3.7"
-  - &UVLOOP
-    stage: test
-    env: UVLOOP="y"
-    python: "3.6"
-  - <<: *UVLOOP
+  # Run examples Python 3.6, 3.7 and system Redis
+  - &EXAMPLES
+    stage: examples
+    python: 3.6
+    env: {}
+    install:
+    - pip install -e. -c tests/requirements.txt
+    script:
+    - make examples
+  - <<: *EXAMPLES
     python: "3.7"
 
 install:
 - make -j ci-build-redis
 - |
     if [ "$UVLOOP" = "y" ]; then
-      export TEST_ARGS="$TEST_ARGS --uvloop"
+      export PYTEST_ADDOPTS="$PYTEST_ADDOPTS --uvloop"
```

Comment:
by the way, there's also `pytest.ini` where you can set up additional options. if you need "xdist" enabled everywhere it'd  work out of the box